### ISSUE TITLE
Incorrect images for "send-backward"

### ISSUE THREAD

**Issue by otrinnes** [FortAwesome/Font-Awesome#15542]
**Description**
There are two images which I consider incorrect:
- far fa-send-backward (https://fontawesome.com/icons/send-backward?style=regular)
- fal fa-send-backward (https://fontawesome.com/icons/send-backward?style=light)

In these images the light box should actually be **behind** the box with the double margin to be consistent with their counter parts "bring-forward". Currently, these images are just rotations of their counter parts. The four images "send-back", "send-backward", "bring-forward", and "bring-front" should all have the a consistent shade for the primary/secondary box, which is not the case for the regular and the light style.

The images for the icons "far fa-send-backward" (solid style) and "fad fa-send-backward" (duotone style) on the other hand are correct. Here, the shade of the primary box is consistently dark.

**To Reproduce**
Apparently I cannot test pro icons on jsfiddle. Just use this code:
```html
<i class="far fa-send-back"></i>
<!-- Incorrect image -->
<i class="far fa-send-backward"></i>
<i class="far fa-bring-forward"></i>
<i class="far fa-bring-front"></i>

<i class="fal fa-send-back"></i>
<!-- Incorrect image -->
<i class="fal fa-send-backward"></i>
<i class="fal fa-bring-forward"></i>
<i class="fal fa-bring-front"></i>
```
You will see that the marked images are incorrect.

**Expected behavior**
The light box should be behind the shaded box, not in front of it.

**Screenshots**
Currently incorrect image:
![image](https://user-images.githubusercontent.com/48085164/64962190-4a511580-d897-11e9-95f0-989b732e13db.png)

Expected image (mock-up):
![image](https://user-images.githubusercontent.com/48085164/64962518-dfeca500-d897-11e9-8669-48ad981e1939.png)

Same goes for the "light" style.

**Version and implementation**
Version: 5.10.2
Browser and version: Google Chrome Version 76.0.3809.132 (Official Build) (64-Bit)

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [x] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

Well, I definitely think there is something weird with those icons (I don't know which is the wrong one, it could be `fas/fad bring-forward`.

Anyway, let's assign @sensibleworld so he can take a look

**Comment by otrinnes** [comment]
Hey, thanks for the quick response!

If we take into account all 16 icons, it becomes clear that the two icons for „send-back“ are the issue. The „bring-forward“ icons are consistent with the „bring-front“ and „send-back“ icons.

Thanks for looking into it!

**Comment by sensibleworld** [comment]
@otrinnes Thanks for the heads-up! After checking these out, it definitely looks like I made a mistake when getting these made. I'll get it fixed up promptly.

**Comment by sensibleworld** [comment]
This should be fixed in the latest 5.11 release.

**Comment by otrinnes** [comment]
> This should be fixed in the latest 5.11 release.

Looking good, thanks!
